tests/test_extractors.py

```diff
@@ -424,13 +426,20 @@ def _check_sortings_equal(self, SX1, SX2):
             self.assertTrue(np.array_equal(train1, train2))
 
     def _check_dumping(self, extractor):
+        # json style
         extractor.dump(file_name='test.json')
         extractor_loaded = se.load_extractor_from_json('test.json')
 
         if 'Recording' in str(type(extractor)):
             self._check_recordings_equal(extractor, extractor_loaded)
         elif 'Sorting' in str(type(extractor)):
             self._check_sortings_equal(extractor, extractor_loaded)
+        
+        
+        # with pickle
```

Comment:
this will fail for all non implemented extractors. Maybe you can make another function and we add it gradually as we add the __init__extractor functions?